### PR TITLE
Removes hard-coded extensions from rml

### PR DESCRIPTION
**Pull request by bmorel**
Those extensions are not useful per se, as if they are absent the engine will search for a list of extensions instead.
Removing the extensions makes things less surprising, preventing one to search for an exact filename which might or not exist.

### CODE REVIEW

**Review by necessarily-equal — APPROVED** [review]

**Comment by necessarily-equal** [comment]
I haven't tested, but looks fine from a code point of view

File: ui/hud_human.rml [hunk 1]
```diff
@@ -70,7 +70,7 @@
 				color:#aaffffff;
 			}
 
-			div.ammo, div.stamina{
```
Comment (illwieckz) [inline ui/hud_human.rml:73]: This one is not extension related. I'm OK with it I just want to be sure it's intentional.

Comment (bmorel) [reply ui/hud_human.rml:73]: This commit should more likely go in another PR actually, since indeed, it's not extension related

Comment (illwieckz) [reply ui/hud_human.rml:73]: Ah sorry, I haven't noticed it was a different commit. It's fine then.

**Pull request merged**